includes/templates/responsive_classic/templates/tpl_modules_products_new_listing.php

```diff
@@ -36,19 +51,19 @@
         $display_products_name = '';
       }
 
-      if (PRODUCT_NEW_LIST_MODEL != '0' and zen_get_show_product_switch($products_new->fields['products_id'], 'model')) {
+      if (PRODUCT_NEW_LIST_MODEL != '0' and $flag_show_model) {
```

Comment:
like the remainder here in the template, do we really need the var?  considering the var is only used once, can we replace the var with the `$config_switches->getSwitch('model')`?  and if we want to use a var, i think the flag_ prefix is not needed.  i would prefer `$show_model`.